setup.py

```diff
@@ -9,6 +9,7 @@
     install_requires=["pandas>=1.2.0"],
     extras_require={
         "dev": ["pytest>=6.2.1"],
+        "ml": ["numba"],
```

Comment:
The different install options should be documented in the repo readme.

Reply:
I put numba in install_requires as I think this should be the default implementation. For future packages that should not be installed as default I will add a readme.